pom.xml

```diff
@@ -62,6 +62,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
   </organization>
 
   <properties>
+    <hadoop2.version>2.7.3</hadoop2.version>
```

Comment:
Nit: This might be a good PR to sort the list :-)

Reply:
Wouldn't it make more sense to do aesthetic changes separately?